### PR TITLE
SDK-39: Java sdk:Add support to create spark cluster

### PR DESCRIPTION
**Pull request by abhisheksr8**
Adding support to create spark clusters..

### CODE REVIEW

File: src/main/java/com/qubole/qds/sdk/java/api/ClusterSparkConfigBuilder.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.qubole.qds.sdk.java.api;
+
+/**
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/api/ClusterSparkConfigBuilder.java]: Move the header to the top.

File: src/main/java/com/qubole/qds/sdk/java/api/ClusterConfigBuilder.java [hunk 2]
```diff
@@ -22,6 +22,10 @@
     public ClusterConfigBuilder state(String state);
 
     public ClusterPrestoConfigBuilder presto_settings();
+    
+    public ClusterSparkConfigBuilder spark_settings();
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/api/ClusterConfigBuilder.java]: I think Java SDK still only has support for v1.2 of the API. And I don't think spark_settings are present in this API version.

Comment (abhisheksr8) [inline src/main/java/com/qubole/qds/sdk/java/api/ClusterConfigBuilder.java]: removed spark settings from this implementation

**Comment by mindprince** [comment]
Squash merged: https://github.com/qubole/qds-sdk-java/commit/99b7b6bb199bc92032059c8790b71797d2f9acfc

**Pull request closed without merging**